checker/tests/run-pass/constant_time_taint.rs

```diff
@@ -39,7 +39,7 @@ pub fn test2(secret: i32) {
 pub fn test3(secret: i32) {
     precondition!(has_tag!(&secret, ConstantTimeTaint));
     let _ = if secret | 99991 < 5 { true } else { false };
-    //~ the branch condition has a ConstantTimeTaintKind tag
+    // ~ the branch condition has a ConstantTimeTaintKind tag
```

Comment:
Di, could you have a look at why this has stopped working?

Reply:
The code generated by Rust is actually constant-time; it optimizes the conditional expression to just `secret | 99991 < 5`.
To get the error message back, we can change it to return non-Boolean values, such as:
``` rust
let _ = if secret | 99991 < 5 { 32767 } else { 10003 };
```